pyro/infer/autoguide/guides.py

```diff
@@ -1382,23 +1382,24 @@ def get_deltas(self, save_params=None):
             elif conditional == "delta":
                 aux_value = zero
             elif conditional == "normal":
-                scale = _deep_getattr(self.scales, name)
                 aux_value = pyro.sample(
                     name + "_aux",
-                    dist.Normal(zero, scale).to_event(1),
+                    dist.Normal(zero, 1).to_event(1),
                     infer={"is_auxiliary": True},
                 )
+                scale = _deep_getattr(self.scales, name)
+                aux_value = aux_value * scale
             elif conditional == "mvn":
                 # This overparametrizes by learning (scale,scale_tril),
                 # enabling faster learning of the more-global scale parameter.
-                scale = _deep_getattr(self.scales, name)
-                scale_tril = _deep_getattr(self.scale_trils, name)
-                scale_tril = scale[..., None] * scale_tril
                 aux_value = pyro.sample(
                     name + "_aux",
-                    dist.MultivariateNormal(zero, scale_tril=scale_tril),
+                    dist.Normal(zero, 1).to_event(1),
                     infer={"is_auxiliary": True},
                 )
+                scale = _deep_getattr(self.scales, name)
+                scale_tril = _deep_getattr(self.scale_trils, name)
+                aux_value = aux_value @ scale_tril.T * scale
```

Comment:
I think you will need to add [logdet](https://github.com/pyro-ppl/pyro/blob/6ec3be8186fb669cdb2dc797920444ba0e5904ca/pyro/distributions/transforms/lower_cholesky_affine.py#L66) of those affine transforms.